frontend/public/components/masthead-toolbar.jsx

```diff
@@ -264,6 +268,13 @@ class MastheadToolbarContents_ extends React.Component {
             externalLink: true,
             href: getOCMLink(clusterID),
             image: <img src={redhatLogoImg} alt="" />,
+            callback: () => {
+              if (this._isMounted) {
+                fireTelemetryEvent('Launcher Menu Accessed', {
```

Comment:
Not sure why you need the `isMounted` check. Seems unnecessary because the action cannot be executed if the component isn't mounted.

Reply:
That was used to test the bug I met before I created `withTelemetry` HOC. I will remove that, thanks for pointing out.